### DISCUSSION TITLE
Hope this can work with markdown tiddlers

### DISCUSSION THREAD

**Discussion by linonetwo** [kookma/TW-Section#14] [Ideas]
<img width="751" alt="截屏2021-11-04 上午1 07 21" src="https://user-images.githubusercontent.com/3746270/140128795-2feefec1-0353-4548-a0ad-b5778b69ae99.png">

I'm using fishing plugin to incremental reading materials from the internet, but web-clipper can only download markdown materials, so I have some md tiddlers in my wiki.

Hope Section editor can work with them too!

**Comment by kookma** [comment]
@linonetwo - unfortunately I do not use MD and not familiar with it too much!
But lets see if volunteer can help!
